apps/finance/test/finance.js

```diff
@@ -39,7 +40,41 @@ contract('Finance App', accounts => {
     let CREATE_PAYMENTS_ROLE, CHANGE_PERIOD_ROLE, CHANGE_BUDGETS_ROLE, EXECUTE_PAYMENTS_ROLE, MANAGE_PAYMENTS_ROLE
     let TRANSFER_ROLE
 
+    // Error strings
+    const errors = makeErrorMappingProxy({
```

Comment:
Is the reason for creating the proxy just throwing if an error hasn't been defined?

Reply:
Yup, was afraid it'd return `undefined` if you mis-typed and then the revert test wouldn't actually be using a reason.